app/components/EntryFormRow/index.js

```diff
@@ -60,36 +20,33 @@ class EntryFormRow extends Component {
         <td>
           <DataSelector
             name="categoryId"
-            value={this.state.categoryId}
+            value={this.props.categoryId}
```

Comment:
Hey, while we are in here making changes, can we declare these props as constants at the top of the method?
So `const { categoryId, description, handleFieldChange } = this.props` etc..